docker-compose.yml

```diff
@@ -24,7 +24,17 @@ services:
     - POSTGRES_DB=${DB_NAME:-nodefactory}
     ports:
     - ${DB_LOCAL_PORT:-9999}:5432
+  swagger:
```

Comment:
I think you are missing command how to run this  in taskfile

Reply:
The docs can be accessed on ```app-url:port/api/swagger``` when the app is ran. Wasn't sure if README.md was the appropriate place to write that down or something else?

Reply:
yep :)

Also, should backend docker image then depend on this?